tutor/commands/config.py

```diff
@@ -220,8 +224,32 @@ def patches_list(context: Context) -> None:
     renderer.print_patches_locations()
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context) -> None:
+    config_file = os.path.join(context.root, "config.yml")
+
+    open_cmd = None
+
+    if which("open"):  # MacOS
+        open_cmd = ["open", config_file]
+    elif which("xdg-open"):  # Linux
+        open_cmd = ["xdg-open", config_file]
+    elif which("start"):  # Windows
+        open_cmd = ["start", '""', config_file]
+    else:
+        click.echo(
```

Comment:
This command should fail if neither open commands are available. You could for instance `raise exceptions.TutorError`.